api/src/services/graphql/index.ts

```diff
@@ -3194,100 +3192,6 @@ export class GraphQLService {
 			});
 		}
 
-		if ('directus_activity' in schema.create.collections) {
```

Comment:
For the same reasoning as stated in https://github.com/directus/directus/pull/22295/files#r1677902904 we should leave these here for now